fuzzers/kirenenko/buildScript/lcms-2017-03-21/buildK.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Can we avoid these redundant build scripts, and use the original `benchmarks/*/build.sh`-s everywhere?

To build the two binaries, consider calling the original build script twice from the fuzzer.py's `build()`, similarly as you're doing it now, with different env vars. To avoid the name conflict, perhaps we could add the `_kir` and `_vani` suffixes by renaming the output file after calling `utils.build_benchmark()` in `build()`?

Feel free to make necessary modifications to the original build scripts when necessary (e.g., add missing `make clean`, or change CXX to CC, use `cp -rf`, etc). That way we won't need three very similar build scripts for each benchmark, only one.

Reply:
Thanks for the note. Are we going to directly modify the build script in each benchmarks directory? Would that impact the other fuzzers and measures? Or do we need to modify the script in build()?

Reply:
> Are we going to directly modify the build script in each benchmarks directory? 

Yes, please.

> Would that impact the other fuzzers and measures? 

Yes, all fuzzers should use the same `build.sh`. Let us know if that causes issues.

Reply:
Thanks for the note. I think I've fixed the requested changes in the new commit.